packages/postcss-minify-gradients/src/isColorStop.js

```diff
@@ -29,9 +29,8 @@ function isCSSLengthUnit(input) {
 }
 
 function isStop(str) {
-  let stop = !str;
```

Comment:
- If `str` is falsy, then `stop` is `true`, so the next branch is skipped and the function returns `true`
- If `str` is truthy then `stop` is `false`, so the function returns the value assigned to `stop` in the `if` branch.